ra-optimisation/search-tree-rao/src/test/java/com/powsybl/openrao/searchtreerao/result/impl/PreventiveAndCurativesRaoResultImplTest.java

```diff
@@ -140,6 +141,9 @@ public void setUp() {
         when(curativeState2.getInstant()).thenReturn(curativeInstant);
         when(curativeState2.getContingency()).thenReturn(Optional.of(contingency2));
         when(curativeState3.getInstant()).thenReturn(curativeInstant);
+        when(crac.getInstantBefore(curativeInstant)).thenReturn(autoInstant);
```

Comment:
at this point, I think we may as well replace all this with an actual crac creation